.envrc

```diff
@@ -1,6 +1,10 @@
 dotenv
+
 if ! has nix_direnv_version || ! nix_direnv_version 3.0.4; then
     source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/3.0.4/direnvrc" "sha256-DzlYZ33mWF/Gs8DDeyjr8mnVmQGx7ASYqA5WlxwvBG4="
 fi
+
+watch_file flake.nix
```

Comment:
This is already handled by the `use_flake` function, but adding it doesn't hurt and makes it more explicit.